pkg/virt-handler/cmd-client/client_test.go

```diff
@@ -228,5 +228,83 @@ var _ = Describe("Virt remote commands", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 		})
+
+		Context("GetGuestInfo", func() {
+			var (
+				mockCmdClient *cmdv1.MockCmdClient
+				client        LauncherClient
+			)
+
+			BeforeEach(func() {
+				ctrl := gomock.NewController(GinkgoT())
+				mockCmdClient = cmdv1.NewMockCmdClient(ctrl)
+				client = newV1Client(mockCmdClient, nil)
+			})
+
+			var (
+				expectGetGuestInfo = func() *gomock.Call {
+					return mockCmdClient.EXPECT().GetGuestInfo(gomock.Any(), &cmdv1.EmptyRequest{})
+				}
+			)
+
+			It("successfully unmarshals guest agent JSON response", func() {
+				guestAgentJSON := `{
```

Comment:
Make it a const

Reply:
can you explain why?

Reply:
I'd say it's best practise. Consider it a nit, but it also doesn't cost you much to change it.

`const guestAgentJSON = ...`